### ISSUE TITLE
🔗 ⛔ SVG element STILL missing accessible name in Icon List widget (Duplicate of #29340)

### ISSUE THREAD

**Issue by aebs90** [elementor/elementor#30867]
### Prerequisites

- [ ] I have searched for similar issues in open and closed tickets and cannot find a duplicate.
- [ ] I have troubleshooted my issue, and it still exists against the latest stable version of Elementor.

### Description

Hello Friends!

I previously reported an issue here: https://github.com/elementor/elementor/issues/30132 and it was marked as _Resolved_ because **you made a fix in Elementor 3.28.2 (Thank you so much!)**

Unfortunately, **the issue that I originally reported still exists**... I understand that the fix from #29340 _sounds_ like a similar issue, but the fix didn't cover this specific scenario:

- SVG
- Without a link (<a>)
- **IN the Icon List widget**

Could you reopen the previous ticket or continue the conversation on this?

I'm not sure what's the right process to follow as I can't comment anymore in the other ticket.

Thanks again for your help!

### Steps to reproduce

Documented on this Ticket: [#30132]

### Expected behavior

Documented on this Ticket: [#30132]

### Isolating the problem

- [ ] This bug happens when only the Elementor (and Elementor Pro) plugins are active.
- [ ] This bug happens with the Hello Elementor theme active.
- [ ] I can reproduce this bug consistently by following the steps I described above.

### Elementor System Info

<details>
<summary>Click to reveal</summary>

````txt

== Server Environment ==
	Operating System: Linux
	Software: nginx
	MySQL version: MariaDB Server v10.11.10-MariaDB-log
	PHP Version: 8.2.28
	PHP Memory Limit: 512M
	PHP Max Input Vars: 6144
	PHP Max Post Size: 2047M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.7.2
	Site URL: (REMOVED)
	Home URL: (REMOVED)
	WP Multisite: No
	Max Upload Size: 2 GB
	Memory limit: 512M
	Max Memory limit: 512M
	Permalink Structure: /%year%/%monthnum%/%day%/%postname%/
	Language: en_US
	Timezone: 0
	Debug Mode: Active

== Theme ==
	Name: Hello Elementor
	Version: 3.3.0
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en-US
	User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/135.0.0.0 Safari/537.36

== Active Plugins ==
	Elementor
		Version: 3.28.3
		Author: Elementor.com

	Elementor Pro
		Version: 3.28.2
		Author: Elementor.com


== Must-Use Plugins ==
	debug-responsive.php
		Version: 
		Author: 

	elementor-find-template-usages.php
		Version: 
		Author: 

	gf-settings.php
		Version: 
		Author: 


== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Inline Font Icons: Active by default
	Additional Custom Breakpoints: Active by default
	Container: Active by default
	Optimized Markup: Inactive by default
	Load Google Fonts locally: Active by default
	Default to New Theme Builder: Active by default
	Header & Footer: Active by default
	Nested Elements: Active by default
	Editor Top Bar: Active by default
	Pages Panel: Inactive by default
	Element Caching: Inactive
	Elementor Home Screen: Active by default
	Elementor Editor Events: Inactive by default
	Atomic Widgets: Inactive by default
	Global Classes: Inactive by default
	Launchpad Checklist: Active by default
	Cloud Library: Inactive by default
	Menu: Inactive by default


== Log ==
	
PHP: showing 5 of 5PHP: 2025-02-05 09:31:13 [warning X 9][../elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php::150] Undefined array key "condition_type" [array (
  'trace' => '
#0: ../elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(150): Elementor\Core\Logger\Manager -> rest_error_handler()
#1: ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_template_json_item()
#2: ../elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(120): class type array_map()
#3: ../elementor-pro/core/app/modules/site-editor/data/endpoints/templates.php(59): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> normalize_templates_json()
#4: ../elementor/data/base/endpoint.php(158): ElementorPro\Core\App\Modules\SiteEditor\Data\Endpoints\Templates -> get_items()
',
)]
PHP: 2025-02-05 09:31:14 [notice X 5][../elementor-pro/modules/screenshots/module.php::33] Function utf8_decode() is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-02-05 09:31:27 [warning X 2][../elementor/includes/managers/image.php::113] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-02-17 14:20:28 [warning X 11][../elementor/includes/conditions.php::87] Undefined array key "product_query_post_type" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2025-03-25 07:38:58 [warning X 6][../elementor/modules/history/revisions-manager.php::363] Undefined array key "editor_post_id" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 352025-03-10 06:44:58 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.27.5',
  'to' => '3.27.6',
)]
2025-03-25 07:32:27 [info] elementor::elementor_updater Started 
2025-03-25 07:32:27 [info] Elementor/Upgrades - _on_each_version Start  
2025-03-25 07:32:27 [info] Elementor/Upgrades - _on_each_version Finished 
2025-03-25 07:32:27 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.27.6',
  'to' => '3.28.1',
)]
2025-03-25 07:32:27 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.27.6',
  'to' => '3.28.1',
)]
2025-03-25 07:37:06 [info] elementor-pro::elementor_pro_updater Started 
2025-03-25 07:37:06 [info] Elementor Pro/Upgrades - _on_each_version Start  
2025-03-25 07:37:06 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2025-03-25 07:37:06 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.27.4',
  'to' => '3.28.1',
)]
2025-04-07 16:25:54 [info] elementor::elementor_updater Started 
2025-04-07 16:25:54 [info] Elementor/Upgrades - _on_each_version Start  
2025-04-07 16:25:54 [info] Elementor/Upgrades - _on_each_version Finished 
2025-04-07 16:25:54 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.28.1',
  'to' => '3.28.3',
)]
2025-04-07 16:25:54 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.28.1',
  'to' => '3.28.3',
)]
2025-04-09 20:21:21 [info] elementor-pro::elementor_pro_updater Started 
2025-04-09 20:21:21 [info] Elementor Pro/Upgrades - _on_each_version Start  
2025-04-09 20:21:21 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2025-04-09 20:21:21 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.28.1',
  'to' => '3.28.2',
)]
2025-04-09 20:21:21 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.28.1',
  'to' => '3.28.2',
)]

JS: showing 3 of 3JS: 2025-02-17 14:20:38 [error X 3][../elementor/assets/js/editor.min.js?ver=3.27.5:3:1310279] can't access property "getBoundingClientRect", L is null 
JS: 2025-02-17 14:25:48 [error X 2][../elementor/assets/js/packages/editor-documents/editor-documents.min.js?ver=3.27.5:1:2370] can't access property "config", t is null 
JS: 2025-04-09 18:16:57 [error X 1][../elementor/assets/js/editor.min.js?ver=3.28.3:3:818562] can't access property "toLowerCase", T.originalEvent.key is undefined 



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==

````

</details>

### Agreement

- [x] I agree that my issue may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hi @aebs90, did you try re-upload ING the file?

**Comment by aebs90** [comment]
@nicholaszein Hi!

Yes, I just tested re-uploading a new SVG and the issue persists.

**Comment by nicholaszein** [comment]
Hello @aebs90,

Thank you for reaching out! 🙏 

### ⛔ While reviewing your report we concluded this is not an issue caused by Elementor. Here's what we found out:
The sanitization for SVG uploads was fixed for all the icon controls in the editor, included in the Icon List widget, and is working as expected. You can see an example of it working in the video below:

https://github.com/user-attachments/assets/71e83956-fb4c-4c3f-982d-156c2fc4cb5c

### ✅ What you can do to solve your issue:
Please keep in mind that the sanitization process happens during the upload process, and cannot be reverted. If you have uploaded an SVG as an icon with aria attributes _**BEFORE**_ the fix we introduced, you won't be able to benefit from the fix. Unfortunately, at this time the only way to fix this is to delete the old SVG icon from your Media Library and upload it again.

### 👍 For now, that's all we can help with. 

If you think this is still an issue caused by Elementor after following the troubleshooting steps above, feel free to open a new 🐞 [Bug Report](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml).

Kind regards
